CA_DataUploaderLib/ICALogger.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace CA_DataUploaderLib
+{
+    public interface ICALogger
```

Comment:
why not just `ILogger` ?

Reply:
used Ilog to not use the same name as Microsoft's one